Packs/PingIdentity/Integrations/PingOne/PingOne.yml

```
@@ -0,0 +1,228 @@
+commonfields:
+  id: PingOne
+  version: -1
+name: PingOne
+display: PingOne
+category: Identity and Access Management
+description: 'Integrates with the PingOne Management API to unlock, create, delete
+  and update users.  '
+configuration:
+- display: Your PingOne Environment ID
+  name: environment_id
+  type: 0
+  required: true
+  additionalinfo: Environment ID is available under the PingOne Dashboard->Environment
+    Properties.
+- display: PingOne Region
+  name: region
+  defaultvalue: US
+  type: 15
+  required: true
+  options:
+  - US
+  - EU
+  - Asia
+  additionalinfo: PingOne has 3 regions, US, EU and Asia.
+- display: Client ID
+  displaypassword: Client Secret
+  name: credentials
+  type: 9
+  required: true
+- display: Trust any certificate (not secure)
+  name: insecure
+  type: 8
+  required: false
+- display: Use system proxy settings
+  name: proxy
+  type: 8
+  required: false
+script:
+  script: ''
+  type: python
+  commands:
+  - name: pingone-unlock-user
+    arguments:
+    - name: username
+      required: true
+      default: true
+      description: Username to unlock.
+    description: Unlock a user's account.
+  - name: pingone-deactivate-user
+    arguments:
+    - name: username
+      required: true
+      default: true
+      description: A username to deactivate.
+    description: Deactivate a user's account.
+  - name: pingone-activate-user
+    arguments:
+    - name: username
+      required: true
+      default: true
+      description: A username to activate.
+    description: Activate a user's account.
+  - name: pingone-set-password
+    arguments:
+    - name: username
+      required: true
+      default: true
+      description: The username to change the password for.
+    - name: password
+      required: true
+      description: The new password.
+    description: Sets a user's password.
+  - name: pingone-add-to-group
+    arguments:
+    - name: username
+      description: 'Username '
+    - name: groupName
+      description: Group Name
+    - name: groupId
+      description: Group ID
+    - name: userId
+      description: User ID
+    description: Add user to the group.
+  - name: pingone-remove-from-group
+    arguments:
+    - name: username
+      description: Username
+    - name: groupName
+      description: Group Name
+    - name: userId
+      description: User ID
+    - name: groupId
+      description: Group ID
+    description: Remove user from the group.
+  - name: pingone-get-groups
+    arguments:
+    - name: username
+      description: Username
```

it should be a mandatory arg, isn’t it?
```suggestion
    - name: username
      description: Username
      required: true
```